substrate/frame/system/src/lib.rs

```diff
@@ -463,6 +467,7 @@ pub mod pallet {
 		type RuntimeTask: Task;
 
 		/// This stores the number of previous transactions associated with a sender account.
+		#[pallet::no_default_bounds]
```

Comment:
Now there are two notations of default, once from the `Default` bound here, and one from the `DefaultNonce`.